src/main/java/pl/marcin/project/runners/ShopAppRunner.java

```diff
@@ -0,0 +1,10 @@
+package pl.marcin.project.runners;
+
+import pl.marcin.project.controller.Shop;
+
+public class ShopAppRunner implements AppRunner {
+    @Override
+    public void runApplication() {
+        Shop.shopRunner();
```

Comment:
this is better! I think, we even don't have to have this method `static` - we can create a "normal" object and run a "normal" method (not `static`) :)  
I'd say, we should go for `static` methods where we don't have a state - so something that operates only on constants, like conversion between miles to km, or calculating surface area of a circle (with `PI` as a constant) etc. Here we have a state... ;) because we are creating several objects within that method :) 
https://stackoverflow.com/questions/2671496/when-to-use-static-methods

Reply:
ok